docs/source/data/kedro_dvc_versioning.md

```diff
@@ -0,0 +1,196 @@
+# Data and pipeline versioning with Kedro and DVC
+
+This document explains how to use [DVC](https://dvc.org/), a command line tool and VS Code Extension to help you develop reproducible machine learning projects, to version datasets and pipelines in your Kedro project.
+
+## Versioning data with .dvc files
+
+### Initialising the repository
+
+For this example, we will be using a Kedro `spaceflights-pandas` starter project, which includes preconfigured datasets and pipelines.
+
+To use DVC as a Python library, install it using `pip` or `conda`, for example:
+`pip install dvc`
+
+Initialise the Kedro project as a git repository: `git init`, and initialise the Kedro project as a DVC repository: `dvc init`. You should see a message such as:
+
+```bash
+Initialized DVC repository.
+
+You can now commit the changes to git.
+
++---------------------------------------------------------------------+
+|                                                                     |
+|        DVC has enabled anonymous aggregate usage analytics.         |
+|     Read the analytics documentation (and how to opt-out) here:     |
+|             <https://dvc.org/doc/user-guide/analytics>              |
+|                                                                     |
++---------------------------------------------------------------------+
+```
+
+### First commits
+
+Suppose you have a dataset in your project, such as:
```

Comment:
There's nothing to suppose, the `spaceflights-pandas` has this dataset right? Maybe reword it as "Verify that your `conf/base/catalog.yml` contains this dataset"

Reply:
Although really this is about `data/01_raw/companies.csv` being present 🤔

Reply:
What about

```suggestion
Verify that your project catalog contains this dataset definition:
```